lab_4_retrieval_w_clustering/start.py

```diff
@@ -45,6 +45,7 @@ def open_files() -> tuple[list[str], list[str]]:
     return (documents, stopwords)
 
 
+
```

Comment:
rm

Reply:
it is a initiation and morivation for coding start.py

Reply:
morivation, ok